src/app/Http/Controllers/Operations/FetchOperation.php

```diff
@@ -60,13 +60,13 @@ private function fetch($arg)
         $config['searchable_attributes'] = $config['searchable_attributes'] ?? $model_instance->identifiableAttribute();
         $config['query'] = isset($config['query']) && is_callable($config['query']) ? $config['query']($config['model']) : $model_instance; // if a closure that has been passed as "query", use the closure - otherwise use the model
 
-        // FetchOperation is aware of an optional parameter 'keys' that will fetch you the entity/entities that match the provided keys
-        if (request()->has('keys')) {
-            $decoded_keys = json_decode(request()->get('keys'));
-            if (is_array($decoded_keys)) {
-                return $model_instance->whereIn($model_instance->getKeyName(), $decoded_keys)->get();
+        // FetchOperation is aware of an optional parameter 'related_keys' that will fetch you the entity/entities that match the provided keys
+        if (request()->has('related_keys')) {
+            $array_keys = explode(',', request()->get('related_keys'));
+            if (is_array($array_keys) && count($array_keys) > 1) {
+                return $model_instance->whereIn($model_instance->getKeyName(), $array_keys)->get();
```

Comment:
`$model_instance->findMany($array_keys)` would probably be shorter here.

Reply:
The problem I see with find is that it returns `->first()` and we want a collection so we need to use `->get()`, because `first()` returns an model instance. 

You can have a look here about what I am talking:

https://github.com/laravel/framework/blob/c4f877e03c28bdea584f062e40025dd5e5be62e8/src/Illuminate/Database/Eloquent/Builder.php#L339

Apart from that findMany() would work great I think.

Reply:
Oh OK! Got it. Best to use ```findMany()``` or keep ```where()``` then - good catch.

Reply:
Update - I left is as it was 😄 ```where``` ftw!